app/views/leagues/show.html.haml

```diff
@@ -33,7 +33,7 @@
 
   - else
     .panel-heading
-      There are no upcoming matches...
+      Your team has no upcoming matches...
```

Comment:
This won't make sense for people not logged in or aren't in a team for this league. It should check whether you're on a team in this league and only then say "Your team has..."